src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasures.test.ts

```diff
@@ -0,0 +1,92 @@
+import { migrateCalculatedMeasures } from "./migrateCalculatedMeasures";
+import { dataModelsForTests } from "@activeviam/data-model-5.0";
+import { contentServer } from "../__test_resources__/contentServer";
+import { uiCalculatedMeasuresFolder } from "../__test_resources__/uiCalculatedMeasuresFolder";
+import { uiDashboardsFolder } from "../__test_resources__/uiDashboardsFolder";
+import { uiWidgetsFolder } from "../__test_resources__/uiWidgetsFolder";
+import _cloneDeep from "lodash/cloneDeep";
+
+const dataModel = dataModelsForTests.sandbox;
+
+const contentServerForTests = _cloneDeep(contentServer);
+
+contentServerForTests.children!.ui.children = {
+  ...contentServerForTests.children!.ui.children,
+  calculated_measures: uiCalculatedMeasuresFolder,
+  dashboards: uiDashboardsFolder,
+  widgets: uiWidgetsFolder,
+};
+migrateCalculatedMeasures(contentServerForTests, dataModel);
+
+describe("migrateCalculatedMeasures", () => {
+  it("Mutates `contentServerForTests` by migrating calculated measures and removing calculated measure definitions from saved widgets and saved dashboards", () => {
+    expect(contentServerForTests).toMatchSnapshot();
```

Comment:
I don't think people would read the snapshot, it's really big.
The smaller unit tests are great and do the job, so let's remove this one.
We could check however that the `ui/calculated_measures` folder gets removed